impl/maven-impl/src/main/java/org/apache/maven/impl/PathModularization.java

```diff
@@ -192,7 +193,7 @@ private PathModularization() {
              * If no descriptor, the "Automatic-Module-Name" manifest attribute is
              * taken as a fallback.
              */
-            try (JarFile jar = new JarFile(path.toFile())) {
+            try (JarFile jar = new JarFile(path.toFile(), false, JarFile.OPEN_READ, target)) {
```

Comment:
When using `JarFile(File)`, the default value for the `verify` argument is `true`.  We're switching to `false`. Is that on purpose ? I don't really think we need signature verification at this point, right ?

Reply:
Yes, the JAR file is opened only for extracting the module name, either from the `module-info.class` or from the `MANIFEST.MF` file. I thought that verifying the JAR file was an overkill in that situation.